src/webgpu/api/validation/texture/destroy.spec.ts

```diff
@@ -3,17 +3,61 @@ Destroying a texture more than once is allowed.
 `;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUConst } from '../../../constants.js';
 import { ValidationTest } from '../validation_test.js';
 
+const descriptor: GPUTextureDescriptor = {
```

Comment:
nit: kDescriptor

But actually I think you can remove this and use `t.getSampledTexture()` or similar.

Reply:
Done.